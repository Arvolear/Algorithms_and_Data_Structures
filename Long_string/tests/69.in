10914464717186206542678140718340
3319967230022200