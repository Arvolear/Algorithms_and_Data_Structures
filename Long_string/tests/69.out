10914464717186209862645370740540
10914464717186203222710910696140
36235665194291724646424557407464914722147148000
3287521822049196